src/components/pages/admin/members/MemberCard.tsx

```
@@ -0,0 +1,80 @@
+          <div className="flex gap-8">
+            <span className="text-14-700 text-gray-100 md:text-15-700">
+              {user.username}
+            </span>
+            <div className="flex items-center gap-4">
+              {user.teams?.map((teamId: string | null) =>
+                teamId ? (
+                  <Badge key={teamId} variant="secondarySmallSquare">
+                    {teamMap[teamId] || "Unknown Team"}
+                  </Badge>
+                ) : (
+                  <span>소속된 팀이 없습니다.</span>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)